packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -32,12 +32,28 @@ class CpuProfilerController
   /// the message that no filters are applied.
   static const userTagNone = 'none';
 
+  /// User tag that marks app start up for Flutter apps.
+  ///
+  /// This needs to match the tag set and unset at these locations,
+  /// respectively:
+  ///
+  /// Flutter Engine - https://github.com/flutter/engine/blob/master/runtime/dart_isolate.cc#L369
```

Comment:
nit: those lines and columns will probably become wrong over time as unrelated lines are added to the files.
Maybe specify a git hash so the link shows the relevant line although perhaps that will be more confusing?
https://github.com/flutter/engine/blob/f52653ba4568b4a40e8624d47773d156740ea9f7/runtime/dart_isolate.cc#L369

Reply:
done